tests/gtests/gtest/CMakeLists.txt

```diff
@@ -3,8 +3,9 @@
 set(TARGET_NAME gtest)
 set(MAIN_SRC "src/gtest-all.cc")
 source_group("" FILES ${MAIN_SRC})
-include_directories(${CMAKE_SOURCE_DIR}/tests/gtests
-                     ${CMAKE_SOURCE_DIR}/tests/gtests/gtest)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+                    ${CMAKE_CURRENT_SOURCE_DIR}/..
+                    ${CMAKE_CURRENT_SOURCE_DIR}/gtest)
```

Comment:
The same as [here](https://github.com/01org/mkl-dnn/pull/44/commits/79159e575e444c846f8b3548b6230041dc14efb9#r110474690): what this line stands for?
`${CMAKE_CURRENT_SOURCE_DIR}/gtest`